FTBFS with Qt 6.7

Building with Qt 6.7 errors out:
```
/build/sddm/src/sddm/data/interfaces/org.freedesktop.login1.Seat.xml:20:5: error: Invalid property specification: 'Invalid'
/build/sddm/src/sddm/data/interfaces/org.freedesktop.login1.Seat.xml:20:68: error: Invalid Interface specification
/build/sddm/src/sddm/data/interfaces/org.freedesktop.login1.Seat.xml:20:68: error: XML error: Unexpected '<'.
make[2]: *** [src/daemon/CMakeFiles/sddm.dir/build.make:127: src/daemon/Login1Seat.cpp] Error 1
```

--- a/data/interfaces/org.freedesktop.login1.Seat.xml
+++ b/data/interfaces/org.freedesktop.login1.Seat.xml
@@ -17,7 +17,7 @@
   <property name="CanGraphical" type="b" access="read">
   </property>
   <property name="Sessions" type="a(so)" access="read">
-   <annotation name="org.qtproject.QtDBus.QtTypeName" value="QList<NamedSessionPath>"/>
+   <annotation name="org.qtproject.QtDBus.QtTypeName" value="QList&lt;NamedSessionPath&gt;"/>
   </property>
   <property name="IdleHint" type="b" access="read">
   </property>

--- a/data/interfaces/org.freedesktop.login1.User.xml
+++ b/data/interfaces/org.freedesktop.login1.User.xml
@@ -33,7 +33,7 @@
    <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
   </property>
   <property name="Sessions" type="a(so)" access="read">
-   <annotation name="org.qtproject.QtDBus.QtTypeName" value="QList<NamedSessionPath>"/>
+   <annotation name="org.qtproject.QtDBus.QtTypeName" value="QList&lt;NamedSessionPath&gt;"/>
   </property>
   <property name="IdleHint" type="b" access="read">
   </property>

